layer.md

```diff
@@ -324,6 +324,9 @@ Any given image is likely to be composed of several of these Image Filesystem Ch
 
 # Non-Distributable Layers
 
+> **NOTE**: Non-distributable layers are deprecated, and not recommended for future use.
+> Clients SHOULD consider the advice below for existing images, but in general SHOULD NOT produce new non-distributable layers.
```

Comment:
Suggest removing "but in general" if we're deprecating. Not sure what client implies here. Are you referring to clients pushing, vs. pulling?

New images, or other artifact types, SHOULD NOT produce non-distributable layers.

Reply:
I've updated the text to be even more terse.

- "Clients" -> "Implementations" for consistency
- Removed the "in general" bit
- Removed redundant text that boils down to "listen to what about the spec is about to say"